app/src/main/java/base/apptrailers/mobile/globant/com/moviesapp/fragments/MovieListFragment.java

```diff
@@ -51,10 +48,38 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         mLayoutManager = new LinearLayoutManager(getContext(), OrientationHelper.VERTICAL, false);
         recyclerView.setLayoutManager(mLayoutManager);
 
-        myDataset = Arrays.asList("Movie1", "Movie2", "Movie3", "Movie4", "Movie5", "Movie6", "Movie7", "Movie8", "Movie9", "Movie10", "Movie11", "Movie12", "Movie13");
-        listAdapter = new MoviesAdapter(myDataset);
-        recyclerView.setAdapter(listAdapter);
-
         return rootView;
     }
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        myDataset = Arrays.asList("Movie1", "Movie2", "Movie3", "Movie4",
+                "Movie5", "Movie6", "Movie7", "Movie8",
+                "Movie9", "Movie10", "Movie11", "Movie12", "Movie13");
+
+        View.OnClickListener listener = new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Bundle bundle = new Bundle();
+
+                bundle.putString("name","Movie");
+                bundle.putString("director","Rober");
+                bundle.putString("year","2015");
+                bundle.putString("gender","Thriller");
+
+                Intent intent = new Intent(getActivity(),MovieDetailActivity.class);
+                intent.putExtra("extra",bundle);
```

Comment:
just as a matter of best practices, create a static method start in the activity and delegate activity creation just for concern purposes